imp/TokenSet.cs

```diff
@@ -176,18 +176,36 @@ private TokenSet()
         } /* end TokenSet */
 
         /* ---------------------------------------------------------------------------
-        * private constructor TokenSet (uint[])
-        * ---------------------------------------------------------------------------
-        * used to instantiate the prefabricated first and follow lists
+         * private TokenSet newFromRawData(uint[])
+         * ---------------------------------------------------------------------------
+         * Used to instantiate the prefabricated first and follow lists.
+         * Expects TSB[segmentCount] to be the number of tokens, and returns null if 
+         * the count does not match.
          * ------------------------------------------------------------------------ */
 

```

Comment:
`from highest to lowest`

do you want me to change my code to run through the segments in reverse, or do you just want the bit values to be printed out in a different order?